director/stat_stress_test.go

```diff
@@ -54,61 +114,149 @@ var (
 func TestStatMemory(t *testing.T) {
 	server_utils.ResetTestState()
 
+	// To allow for developer control over testing, we add two environment variables
+	// specific to this unit test:
+	// - PELICAN_STRESS_DURATION: Parsed as a time duration, this is the time of the
```

Comment:
Not a change request, but a note for posterity -- these env vars will trigger a warning related to unrecognized Pelican configuration because the client init a few lines down matches all `PELICAN_` env vars and tries to marshal them into the config structs. I think this is fine for an internal unit test.